WordPress/Tests/XSS/EscapeOutputUnitTest.inc

```diff
@@ -205,3 +205,13 @@ to_screen( esc_form_field( $var1), esc_attr( $var2 ) ); // Ok.
 echo esc_form_field( $var ); // Bad.
 echo post_info( $post_id, 'field' ); // Bad.
 echo cpt_info( $post_type, 'query' ); // Bad.
+
+// Bad.
+?>
+<?= $var ?>
+<?php
+
+// Ok.
+<?= esc_html( $var ); ?>
```

Comment:
Seeing the build failure: you could just change the comment to an HTML comment instead ?